java/benchmarks/src/main/java/glide/benchmarks/clients/glide/GlideAsyncClient.java

```diff
@@ -0,0 +1,61 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.benchmarks.clients.glide;
+
+import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import glide.benchmarks.clients.AsyncClient;
+import glide.benchmarks.utils.ConnectionSettings;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/** A Glide client with async capabilities */
+public class GlideAsyncClient implements AsyncClient<String> {
+    private RedisClient redisClient;
```

Comment:
```suggestion
    private BaseClient redisClient;
```
So it could be cluster or standalone

Reply:
The set/get code will be different.  So I had to make them separate clients.

Reply:
why different?

Reply:
Oh because `customCommand` is used there. I think you can merge clients after #919